proposals/target-typed-new.md

```diff
@@ -76,13 +68,12 @@ Action a = new(); // no constructor found
 
 ### Miscellaneous
 
-`throw new()` is allowed.
-
-Target-typed `new` is not allowed with binary operators.
-
-It is disallowed when there is no type to target: unary operators, collection of a `foreach`, in a `using`, in a deconstruction, in an `await` expression, as an anonymous type property (`new { Prop = new() }`), in a `lock` statement, in a `sizeof`, in a `fixed` statement, in a member access (`new().field`), in a dynamically dispatched operation (`someDynamic.Method(new())`), in a LINQ query, as the operand of the `is` operator, as the left operand of the `??` operator,  ...
+The following are consequences of the specification:
```

Comment:
We specifically do not permit to target unspeakable types i.e. anonymous types, I think that needs to be specified as it's not a consequence of the above.

Reply:
There is no need.  The conversion exists but the creation expression is an error.

---
In reply to: [397953799](https://github.com/dotnet/csharplang/pull/3310#discussion_r397953799) [](ancestors = 397953799)